huaweicloud/services/sfs/resource_huaweicloud_sfs_turbo_obs_target.go

```diff
@@ -0,0 +1,322 @@
+func ResourceOBSTarget() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceOBSTargetCreate,
+		ReadContext:   resourceOBSTargetRead,
+		DeleteContext: resourceOBSTargetDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceOBSTargetImportState,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"share_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"file_system_path": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"obs": {
+				Type:     schema.TypeList,
+				Required: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem:     obsSchema(),
+			},
+			"delete_data_in_file_system": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func obsSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"bucket": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"endpoint": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+	return &sc
+}
+
+func buildCreateOBSTargetBodyParams(d *schema.ResourceData) map[string]interface{} {
+	bodyParams := map[string]interface{}{
+		"file_system_path": d.Get("file_system_path"),
+		"obs":              buildOBSBody(d.Get("obs.0").(map[string]interface{})),
+	}
+	return bodyParams
+}
+
+func buildOBSBody(obsData map[string]interface{}) map[string]interface{} {
+	return map[string]interface{}{
+		"bucket":   obsData["bucket"],
+		"endpoint": obsData["endpoint"],
+	}
+}
+
+func resourceOBSTargetCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+	client, err := cfg.SfsV1Client(region)
+	if err != nil {
+		return diag.Errorf("error creating SFS v1 client: %s", err)
+	}
+
+	createObsTargetHttpUrl := "sfs-turbo/shares/{share_id}/targets"
+	createObsTargetPath := client.ResourceBaseURL() + createObsTargetHttpUrl
+	createObsTargetPath = strings.ReplaceAll(createObsTargetPath, "{share_id}", d.Get("share_id").(string))
+
+	createObsTargetOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			202,
+		},
+	}
+	createObsTargetOpt.JSONBody = utils.RemoveNil(buildCreateOBSTargetBodyParams(d))
+	createObsTargetResp, err := client.Request("POST", createObsTargetPath, &createObsTargetOpt)
+	if err != nil {
+		return diag.Errorf("error creating SFS Turbo OBS target: %s", err)
```

Comment:
```suggestion
		return diag.Errorf("error creating OBS target to the SFS Turbo: %s", err)
```
please check the other places

Reply:
done